src/index.html

```diff
@@ -1,13 +1,459 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>The MET</title>
+    <link
+    rel="shortcut icon"
+    href="./images/icons/the_met_favicon.jpg"
+    type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Landing Page</h1>
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo"></a>
+        <div class="top-bar__icons">
+          <a
+          href="#"
+          class="
+          icon
+          icon--phone"
+          >
+          </a>
+          <!-- Hover menu on header -->
+          <div class="tooltip">
+            <p class="tooltip__address">The Met Fifth Avenue</p>
+            <a
+            href="tel:+1 212-535-7710"class="tooltip__phone">
+              +1 212-535-7710
```

Comment:
Need to fix format number in href and fix everywhere your code with code style
```suggestion
            <a
              href="tel:+12125357710"
              class="tooltip__phone"
            >
              +1 212-535-7710
```